### PR TITLE
feat(Validator): add uppercase validator

### PR DESCRIPTION
**Pull request by atlwendy**

### CODE REVIEW

File: terminus-ui/src/services/validators/validators/uppercase/uppercase.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  ValidatorFn,
+  AbstractControl,
+  ValidationErrors,
+} from '@angular/forms';
+import { createContainsUppercaseRegex } from '@terminus/ngx-tools/regex';
+
+
+/**
+ * Return a validator function to verify that number of uppercase letters is satisfied
+ *
```
Comment (benjamincharity) [inline terminus-ui/src/services/validators/validators/uppercase/uppercase.ts]: Missing JSDoc `@param`

**Comment by benjamincharity** [comment]
@atlwendy I think  a couple test cases may have been swapped around. One is failing that shouldn't, and one isn't that should.

**Comment by atlwendy** [comment]
@benjamincharity I don't know how that happened. I checked the tests before commit and was sure they were good. maybe it's because of `out-tsc` directory issue? Anyway, I removed that directory and should be good now. Please re review.

**Pull request merged**